src/controller/java/templates/ChipStructs-java.zapt

```diff
@@ -0,0 +1,58 @@
+{{> header}}
+{{#if (chip_has_client_clusters)}}
```

Comment:
why do we need this top level if?

What happens if no client clusters? would javac accept a completely empty file? Should we not have at least a completely empty ChipStructs class?